OBAKit/Helpers/OBACommonV1.h

```diff
@@ -16,6 +16,9 @@
 
 @import Foundation;
 
+#ifndef _OBA_COMMON_V1_
+#define _OBA_COMMON_V1_
+
```

Comment:
I didn't think they were either, but the compiler started complaining about double inclusion of this file and I added these guards in a fit of annoyance.